packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -120,6 +499,19 @@ class ListWheelScrollView extends StatelessWidget {
 ///  * [RenderListWheelViewport], the render object that renders the children
 ///    on a wheel.
 class ListWheelViewport extends MultiChildRenderObjectWidget {
+  /// Create a viewport where children are rendered onto a wheel.
+  ///
+  /// [diameterRatio] defaults to 2.0 and must not be null.
+  ///
+  /// [perspective] defaults to 0.003 and must not be null.
+  ///
+  /// [itemExtent] in pixels must be provided and must be positive.
+  ///
+  /// [clipToSize] defaults to true and must not be null.
+  ///
+  /// [renderChildrenOutsideViewport] defaults to false and must not be null.
+  ///
+  /// [offset] must be provided and must not be null.
```

Comment:
This is fine, but generally speaking, prefer to not start sentences with a lowercase letter. e.g. "The [foo] argument must...".

Reply:
Done